### PR TITLE
Remove "oakland" from table names.

### PR DESCRIPTION
**Pull request by mikeubell**

### CODE REVIEW

File: bin/export-test-case [hunk 1]
```diff
@@ -7,8 +7,8 @@ set -euo pipefail
 # 2. Fill this section out:
 test_case_dir=spec/fixtures/referendum_supporters_without_expenditures_are_included
 
-# 3. Set up your test file such that any objects from the "oakland_"
-# spreadsheets (oakland_candidate, oakland_committee, oakland_name_to_number,
+# 3. Set up your test file such that any objects from the ""
```
[the remark this answers is not in the document]
Comment (mikeubell) [reply bin/export-test-case]: Travis seemed to be telling me about the models...

**Comment by tdooner** [comment]
<details>
<summary>Build diff from Commit 3b3524de02bc55f88e2c1c58fbd31cc98e52d64c:</summary>

```diff
diff --git a/build/_candidates/oakland/2018-11-06/aimee-eng.md b/build/_candidates/oakland/2018-11-06/aimee-eng.md
index 803472e..05d6941 100644
--- a/build/_candidates/oakland/2018-11-06/aimee-eng.md
+++ b/build/_candidates/oakland/2018-11-06/aimee-eng.md
@@ -1,10 +1,6 @@
 ---
 election: _elections/oakland/2018-11-06.md
 committee_name: Aimee Eng for School Board 2018
-data_warning: This data reflects information from 2018 and prior campaign filings
-  because the candidate is using the same campaign account for multiple elections.
-  Therefore contribution and expenditure calculations may include money raised for
-  or spent on prior elections.
 filer_id: '1368984'
 is_accepted_expenditure_ceiling: true
 is_incumbent: true
diff --git a/build/_candidates/oakland/2018-11-06/desley-brooks.md b/build/_candidates/oakland/2018-11-06/desley-brooks.md
index f40c887..0cf6ec9 100644
--- a/build/_candidates/oakland/2018-11-06/desley-brooks.md
+++ b/build/_candidates/oakland/2018-11-06/desley-brooks.md
@@ -1,10 +1,6 @@
 ---
 election: _elections/oakland/2018-11-06.md
 committee_name: Desley Brooks for City Council 2018
-data_warning: This data reflects information from 2018 and prior campaign filings
-  because the candidate is using the same campaign account for multiple elections.
-  Therefore contribution and expenditure calculations may include money raised for
-  or spent on prior elections.
 filer_id: '1236617'
 is_accepted_expenditure_ceiling: true
 is_incumbent: true
diff --git a/build/_committees/1236617.md b/build/_committees/1236617.md
index f306908..25cc62a 100644
--- a/build/_committees/1236617.md
+++ b/build/_committees/1236617.md
@@ -3,8 +3,5 @@ filer_id: '1236617'
 name: Desley Brooks for City Council 2018
 candidate_controlled_id: ''
 title: Desley Brooks for City Council 2018
-data_warning: This data reflects information from 2018 and prior campaign filings
-  because the candidate is using the same campaign account for multiple elections.
-  Therefore contribution and expenditure calculations may include money raised for
-  or spent on prior elections.
+data_warning: 
 ---
diff --git a/build/_committees/1368984.md b/build/_committees/1368984.md
index 26aca64..760fbf6 100644
--- a/build/_committees/1368984.md
+++ b/build/_committees/1368984.md
@@ -3,8 +3,5 @@ filer_id: '1368984'
 name: Aimee Eng for School Board 2018
 candidate_controlled_id: ''
 title: Aimee Eng for School Board 2018
-data_warning: This data reflects information from 2018 and prior campaign filings
-  because the candidate is using the same campaign account for multiple elections.
-  Therefore contribution and expenditure calculations may include money raised for
-  or spent on prior elections.
+data_warning: 
 ---
diff --git a/build/_data/candidates/oakland/2018-11-06/cat-brooks.json b/build/_data/candidates/oakland/2018-11-06/cat-brooks.json
index 1ba3552..3ad62e5 100644
--- a/build/_data/candidates/oakland/2018-11-06/cat-brooks.json
+++ b/build/_data/candidates/oakland/2018-11-06/cat-brooks.json
@@ -1,5 +1,5 @@
 {
-  "id": 12,
+  "id": 54,
   "name": "Cat Brooks",
   "photo_url": "Cat-Brooks.png",
   "website_url": "https://www.catbrooksforoakland.com/",
diff --git a/build/_data/candidates/oakland/2018-11-06/mya-whitaker.json b/build/_data/candidates/oakland/2018-11-06/mya-whitaker.json
index e4fccc0..991f285 100644
--- a/build/_data/candidates/oakland/2018-11-06/mya-whitaker.json
+++ b/build/_data/candidates/oakland/2018-11-06/mya-whitaker.json
@@ -1,5 +1,5 @@
 {
-  "id": 11,
+  "id": 53,
   "name": "Mya Whitaker",
   "photo_url": "Mya-Whitaker.png",
   "website_url": "https://whitakerforoakland.com",
diff --git a/build/_data/candidates/oakland/2018-11-06/natasha-middleton.json b/build/_data/candidates/oakland/2018-11-06/natasha-middleton.json
index 76f0da2..6f1f7dc 100644
--- a/build/_data/candidates/oakland/2018-11-06/natasha-middleton.json
+++ b/build/_data/candidates/oakland/2018-11-06/natasha-middleton.json
@@ -1,5 +1,5 @@
 {
-  "id": 27,
+  "id": 11,
   "name": "Natasha Middleton",
   "photo_url": "Natasha-Middleton.png",
   "website_url": "https://www.natashaforoakland.com/",
diff --git a/build/_data/candidates/oakland/2018-11-06/nayeli-maxson.json b/build/_data/candidates/oakland/2018-11-06/nayeli-maxson.json
index 26b261d..0444d56 100644
--- a/build/_data/candidates/oakland/2018-11-06/nayeli-maxson.json
+++ b/build/_data/candidates/oakland/2018-11-06/nayeli-maxson.json
@@ -1,5 +1,5 @@
 {
-  "id": 35,
+  "id": 12,
   "name": "Nayeli Maxson",
   "photo_url": "Nayeli-Maxson.png",
   "website_url": "https://nayeliforoakland.com/",
diff --git a/build/_data/candidates/oakland/2018-11-06/nikki-fortunato-bas.json b/build/_data/candidates/oakland/2018-11-06/nikki-fortunato-bas.json
index 9d15326..1801322 100644
--- a/build/_data/candidates/oakland/2018-11-06/nikki-fortunato-bas.json
+++ b/build/_data/candidates/oakland/2018-11-06/nikki-fortunato-bas.json
@@ -1,5 +1,5 @@
 {
-  "id": 53,
+  "id": 27,
   "name": "Nikki Fortunato Bas",
   "photo_url": "nikki_fortunato_bas.png",
   "website_url": "http://www.nikki4oakland.com",
diff --git a/build/_data/candidates/oakland/2018-11-06/pamela-price.json b/build/_data/candidates/oakland/2018-11-06/pamela-price.json
index 0de26b4..3765ffe 100644
--- a/build/_data/candidates/oakland/2018-11-06/pamela-price.json
+++ b/build/_data/candidates/oakland/2018-11-06/pamela-price.json
@@ -1,5 +1,5 @@
 {
-  "id": 54,
+  "id": 35,
   "name": "Pamela Price",
   "photo_url": "Pamela-Price.png",
   "website_url": "https://www.pamelaprice4mayor.com/",
```
</details>

**Comment by tdooner** [comment]
<details>
<summary>Build diff from Commit 16c7b08e9d9c5d42ddaebf14f2ef4440fd7c1041:</summary>

```diff
diff --git a/build/_candidates/oakland/2018-11-06/aimee-eng.md b/build/_candidates/oakland/2018-11-06/aimee-eng.md
index 803472e..05d6941 100644
--- a/build/_candidates/oakland/2018-11-06/aimee-eng.md
+++ b/build/_candidates/oakland/2018-11-06/aimee-eng.md
@@ -1,10 +1,6 @@
 ---
 election: _elections/oakland/2018-11-06.md
 committee_name: Aimee Eng for School Board 2018
-data_warning: This data reflects information from 2018 and prior campaign filings
-  because the candidate is using the same campaign account for multiple elections.
-  Therefore contribution and expenditure calculations may include money raised for
-  or spent on prior elections.
 filer_id: '1368984'
 is_accepted_expenditure_ceiling: true
 is_incumbent: true
diff --git a/build/_candidates/oakland/2018-11-06/desley-brooks.md b/build/_candidates/oakland/2018-11-06/desley-brooks.md
index f40c887..0cf6ec9 100644
--- a/build/_candidates/oakland/2018-11-06/desley-brooks.md
+++ b/build/_candidates/oakland/2018-11-06/desley-brooks.md
@@ -1,10 +1,6 @@
 ---
 election: _elections/oakland/2018-11-06.md
 committee_name: Desley Brooks for City Council 2018
-data_warning: This data reflects information from 2018 and prior campaign filings
-  because the candidate is using the same campaign account for multiple elections.
-  Therefore contribution and expenditure calculations may include money raised for
-  or spent on prior elections.
 filer_id: '1236617'
 is_accepted_expenditure_ceiling: true
 is_incumbent: true
diff --git a/build/_committees/1236617.md b/build/_committees/1236617.md
index f306908..25cc62a 100644
--- a/build/_committees/1236617.md
+++ b/build/_committees/1236617.md
@@ -3,8 +3,5 @@ filer_id: '1236617'
 name: Desley Brooks for City Council 2018
 candidate_controlled_id: ''
 title: Desley Brooks for City Council 2018
-data_warning: This data reflects information from 2018 and prior campaign filings
-  because the candidate is using the same campaign account for multiple elections.
-  Therefore contribution and expenditure calculations may include money raised for
-  or spent on prior elections.
+data_warning: 
 ---
diff --git a/build/_committees/1368984.md b/build/_committees/1368984.md
index 26aca64..760fbf6 100644
--- a/build/_committees/1368984.md
+++ b/build/_committees/1368984.md
@@ -3,8 +3,5 @@ filer_id: '1368984'
 name: Aimee Eng for School Board 2018
 candidate_controlled_id: ''
 title: Aimee Eng for School Board 2018
-data_warning: This data reflects information from 2018 and prior campaign filings
-  because the candidate is using the same campaign account for multiple elections.
-  Therefore contribution and expenditure calculations may include money raised for
-  or spent on prior elections.
+data_warning: 
 ---
diff --git a/build/_data/candidates/oakland/2018-11-06/cat-brooks.json b/build/_data/candidates/oakland/2018-11-06/cat-brooks.json
index 1ba3552..3ad62e5 100644
--- a/build/_data/candidates/oakland/2018-11-06/cat-brooks.json
+++ b/build/_data/candidates/oakland/2018-11-06/cat-brooks.json
@@ -1,5 +1,5 @@
 {
-  "id": 12,
+  "id": 54,
   "name": "Cat Brooks",
   "photo_url": "Cat-Brooks.png",
   "website_url": "https://www.catbrooksforoakland.com/",
diff --git a/build/_data/candidates/oakland/2018-11-06/mya-whitaker.json b/build/_data/candidates/oakland/2018-11-06/mya-whitaker.json
index e4fccc0..991f285 100644
--- a/build/_data/candidates/oakland/2018-11-06/mya-whitaker.json
+++ b/build/_data/candidates/oakland/2018-11-06/mya-whitaker.json
@@ -1,5 +1,5 @@
 {
-  "id": 11,
+  "id": 53,
   "name": "Mya Whitaker",
   "photo_url": "Mya-Whitaker.png",
   "website_url": "https://whitakerforoakland.com",
diff --git a/build/_data/candidates/oakland/2018-11-06/natasha-middleton.json b/build/_data/candidates/oakland/2018-11-06/natasha-middleton.json
index 76f0da2..6f1f7dc 100644
--- a/build/_data/candidates/oakland/2018-11-06/natasha-middleton.json
+++ b/build/_data/candidates/oakland/2018-11-06/natasha-middleton.json
@@ -1,5 +1,5 @@
 {
-  "id": 27,
+  "id": 11,
   "name": "Natasha Middleton",
   "photo_url": "Natasha-Middleton.png",
   "website_url": "https://www.natashaforoakland.com/",
diff --git a/build/_data/candidates/oakland/2018-11-06/nayeli-maxson.json b/build/_data/candidates/oakland/2018-11-06/nayeli-maxson.json
index 26b261d..0444d56 100644
--- a/build/_data/candidates/oakland/2018-11-06/nayeli-maxson.json
+++ b/build/_data/candidates/oakland/2018-11-06/nayeli-maxson.json
@@ -1,5 +1,5 @@
 {
-  "id": 35,
+  "id": 12,
   "name": "Nayeli Maxson",
   "photo_url": "Nayeli-Maxson.png",
   "website_url": "https://nayeliforoakland.com/",
diff --git a/build/_data/candidates/oakland/2018-11-06/nikki-fortunato-bas.json b/build/_data/candidates/oakland/2018-11-06/nikki-fortunato-bas.json
index 9d15326..1801322 100644
--- a/build/_data/candidates/oakland/2018-11-06/nikki-fortunato-bas.json
+++ b/build/_data/candidates/oakland/2018-11-06/nikki-fortunato-bas.json
@@ -1,5 +1,5 @@
 {
-  "id": 53,
+  "id": 27,
   "name": "Nikki Fortunato Bas",
   "photo_url": "nikki_fortunato_bas.png",
   "website_url": "http://www.nikki4oakland.com",
diff --git a/build/_data/candidates/oakland/2018-11-06/pamela-price.json b/build/_data/candidates/oakland/2018-11-06/pamela-price.json
index 0de26b4..3765ffe 100644
--- a/build/_data/candidates/oakland/2018-11-06/pamela-price.json
+++ b/build/_data/candidates/oakland/2018-11-06/pamela-price.json
@@ -1,5 +1,5 @@
 {
-  "id": 54,
+  "id": 35,
   "name": "Pamela Price",
   "photo_url": "Pamela-Price.png",
   "website_url": "https://www.pamelaprice4mayor.com/",
```
</details>

**Comment by tdooner** [comment]
<details>
<summary>Build diff from Commit ab4f5209144d77311258329aed605023cb665eae:</summary>

```diff

```
</details>

**Pull request merged**